### PR TITLE
Update documentation for Package Manager 1.3 client

### LINKED ISSUE #3969: Add no parameter documentation
This page is quite well detailed, but it omits `winget upgrade` with no parameter.  The omission of the parameter lists all packages with upgrades available, which I contend is not very discoverable as all other upgrade commands and parameters have to do with the performance of an upgrade.  I had feared it would behave like `winget upgrade --all` and it wasn't until I saw a Stack Overflow answer that I even tried it.  When I looked at this page, I saw no mention of it, so I hope to see it get added to the docs.

The section of using `winget list` in conjunction with `winget upgrade` to find and enact upgrades should also mention this improved strategy.  `winget list` is still important as `winget upgrade` only shows available upgrades. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2cf54ba9-53bd-e5d0-3b5a-e5b003f356d9
* Version Independent ID: ed2c403c-7bba-9647-fb5c-da556cb93a6b
* Content: [upgrade Command](https://docs.microsoft.com/en-us/windows/package-manager/winget/upgrade#feedback)
* Content Source: [hub/package-manager/winget/upgrade.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/hub/package-manager/winget/upgrade.md)
* Product: **dev-environment**
* Technology: **package-manager**
* GitHub Login: @KevinLaMS
* Microsoft Alias: **kevinla**

### PR DESCRIPTION
**Pull request by ryfu-msft**
Fixes #3969 

Updates the Package Manager docs with changes from the latest 1.3 client.

### CODE REVIEW

**Comment by PRMerger-2** [comment]
@ryfu-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by jborsecnik** [comment]
@KevinLaMS, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by PRMerger7** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/package-manager/winget/upgrade.md [hunk 1]
```diff
@@ -75,11 +76,11 @@ The following example shows upgrading all apps
 winget upgrade --all
 ```
 
-## Using list and upgrade
+## Using **upgrade**
 
-It is common to use the [**list**](.\list.md) command to identify apps in need of an update, and then to use **upgrade** to install the latest.
+To identify which apps are in need of an update, simply use **upgrade** without any arguments to show all available upgrades.
```
Comment (mattwojo) [inline hub/package-manager/winget/upgrade.md:81]: Is list no longer supported?

**Comment by mattwojo** [comment]
@ryfu-msft - It looks like this PR will update the issue this customer filed: https://github.com/MicrosoftDocs/windows-uwp/issues/3969 ..but can you take a look to confirm?

**Comment by PRMerger5** [comment]
@ryfu-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/package-manager/winget/upgrade.md (hunk 1 above)
Comment (ryfu-msft) [reply hub/package-manager/winget/upgrade.md:81]: List is still supported. I added back the description and example for how to use the list command with upgrade as I feel that it is still useful and relevant for identifying an update for a specific package.

**Comment by ryfu-msft** [comment]
> @ryfu-msft - It looks like this PR will update the issue this customer filed: #3969 ..but can you take a look to confirm?

Yes, I believe this PR will resolve issue 3969 as my changes now include documentation for *winget upgrade* . I went ahead and linked that issue to this PR.

**Comment by ryfu-msft** [comment]
@denelon, I updated the settings file json example in the settings.md file as well as documented that experimental features are only available in preview builds.

**Review by denelon — APPROVED** [review]

**Comment by mattwojo** [comment]
Looks good!

**Pull request merged**